### DISCUSSION TITLE
Add Grain framerate control

### DISCUSSION THREAD

**Discussion by Xaryen** [videovillage/community#177] [Ideas]
(filmbox, I don't seem to be able to add a label)
Hello, it would be nice to have the option to set the frame rate or "posterize time" of the grain, i.e. to set it to a fraction of the footage framerate. It can be useful for stylization as well as appeasing the compression gods.

**Comment by gregcotten** [comment]
Interesting - I have never heard of anyone trying to mitigate compression issues of grain by "holding" the grain pattern for multiple frames.

**Reply by Xaryen** [reply]
It can be quite useful method when working with a lot of still images (e.g. in 2D animation) where setting the grain on "2's", "3's" or just making it static can help a bit, though I'd understand if it's a bit too much of a niche case.

**Reply by gregcotten** [reply]
makes sense to me - I'll try and add in support for this
